packages/@aws-cdk/applet-js/bin/cdk-applet-js.ts

```diff
@@ -81,7 +81,7 @@ async function constructStack(app: cdk.App, searchDir: string, tempDir: string,
 
   // we need to resolve the module name relatively to where the applet file is
   // and not relative to this module or cwd.
-  const modulePath = require.resolve(applet.moduleName, { paths: [ searchDir ] });
+  const modulePath = path.resolve(searchDir, applet.moduleName);
```

Comment:
Will this still work if the moduleName is in a `node_modules` directory? `path.resolve()` won't find it then.

Reply:
Ugh. It probably won't. But then again... resolving relative to `process.cwd()` does _strictly not_ work with node 12 😭

Reply:
I think the problem is that `require.resolve()` just does not do what I thought it does. It needs a list of search paths (in effect, `node_modules` paths plus current directory) to which it will append the module name in order.

There's no easy way to build a set of standardized NodeJS search paths from a directory name though--we have to do that ourselves (by recursing up the tree and adding a `node_modules` directory to each).

Reply:
I decided to revert this change - it seems to be a bug in node, so I filed nodejs/node#27794 instead of trying to work around.